packages/cli/src/handlers/download.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-await-in-loop */
+import {
+    ApiChartAsCodeListResponse,
+    ApiChartAsCodeUpsertResponse,
+    AuthorizationError,
+    SavedChart,
+} from '@lightdash/common';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import { getConfig } from '../config';
+import GlobalState from '../globalState';
+import { checkLightdashVersion, lightdashApi } from './dbt/apiClient';
+
+const DOWNLOAD_FOLDER = 'target';
+export type DownloadHandlerOptions = {
+    verbose: boolean;
+};
+export const downloadHandler = async (
+    options: DownloadHandlerOptions,
+): Promise<void> => {
+    GlobalState.setVerbose(options.verbose);
+    await checkLightdashVersion();
+
+    const config = await getConfig();
+    if (!config.context?.apiKey || !config.context.serverUrl) {
+        throw new AuthorizationError(
+            `Not logged in. Run 'lightdash login --help'`,
+        );
+    }
+
+    const projectId = config.context.project;
+    if (!projectId) {
+        throw new Error(
+            'No project selected. Run lightdash config set-project',
+        );
+    }
+    const chartsAsCode = await lightdashApi<
+        ApiChartAsCodeListResponse['results']
+    >({
+        method: 'GET',
+        url: `/api/v1/projects/${projectId}/coder/charts`,
+        body: undefined,
+    });
+    console.info(`Downloading ${chartsAsCode.length} charts`);
+
+    const outputDir = path.join(process.cwd(), DOWNLOAD_FOLDER);
+    console.info(`Creating new path for files on ${outputDir} `);
+
+    try {
+        await fs.mkdir(outputDir, { recursive: true });
+    } catch (error) {
+        // Directory already exists
+    }
+
+    for (const chart of chartsAsCode) {
+        const chartPath = path.join(outputDir, `${chart.slug}.json`);
+        GlobalState.debug(`> Writing chart to ${chartPath}`);
+        await fs.writeFile(chartPath, JSON.stringify(chart, null, 2));
```

Comment:
let's serialize as yml now, it's more consistent with a dbt project

Reply:
Done

Reply:
We could add in the future a --json format on the donwload command to allow both formats